server/eclipse-project/src/main/java/us/freeandfair/corla/Main.java

```diff
@@ -224,6 +224,10 @@ public void start() {
     // provide properties to the persistence engine
     Persistence.setProperties(my_properties);
 
+    if (!Persistence.hasDB()) {
```

Comment:
What is “the top-level”? If this particular error happens, the server doesn’t start.